idd/Energy+.idd.in

```diff
@@ -99859,6 +99883,7 @@ Output:EnvironmentalImpactFactors,
        \key Timestep
        \key Hourly
        \key Daily
+       \key Annual
```

Comment:
Added environment and runperiod.  Source code currently ignores detailed without error and does not set up the meters.